examples/apexlog/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   363389,                             // use it as an environment variable
+		ProjectKey:  "baa755018d5e35e07897ee0087fcce9c", // use it as an environment variable
+		Environment: "production",                       // use it as an environment variable
+	})
+
+	// Close the airbrake notifier
+	defer airbrake.Close()
+
+	// Create the airbrake handler for "github.com/apex/log"
+	// Insert the airbrake notifier instance and set the severity level.
+	// The acceptable severity levels are "debug" "info", "warning", "error", and "fatal".
+	// If the severity level is not recognized, "error" severity is used.
+	airbrakeHandler := apexlog.New(airbrake, "error")
```

Comment:
We shouldn't accepts strings, it should be `severity` so that you don't validate input during runtime.

```suggestion
	airbrakeHandler := apexlog.New(airbrake, apexlog.ErrorLevel)
```

Reply:
We still pass a string instead of `severity`.